superset-frontend/src/pages/ChartList/index.tsx

```diff
@@ -157,6 +157,20 @@ const StyledActions = styled.div`
   color: ${({ theme }) => theme.colors.grayscale.base};
 `;
 
+const DashboardCrossLinks = React.memo(
```

Comment:
It appears to be more beneficial to create a dedicated file specifically for this component.

Reply:
Agreed 👍

Reply:
done